src/helpers.rs

```diff
@@ -43,6 +43,26 @@ const UNIX_IO_ERROR_TABLE: &[(std::io::ErrorKind, &str)] = {
         (WouldBlock, "EWOULDBLOCK"),
```

Comment:
Please add a comment above this table explaining that it should be the same as `decode_error_kind` in https://github.com/rust-lang/rust/blob/master/library/std/src/sys/unix/mod.rs.